File: bank-back/src/main/java/com/fuzzysets/bank/dao/BankDto.java
package com.fuzzysets.bank.dao;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.experimental.FieldDefaults;

@Data
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class BankDto {
    String name;
    Integer target;
    Integer months;
    Integer risk;
    Integer percent;
}

File: bank-front/src/app/models/BankDto.ts
export class BankDto {
    name: string | null;
    target: number;
    months: number;
    risk: number;
    percent: number;
  
    constructor() {
      this.name = null;
      this.target = 0;
      this.months = 0;
      this.risk = 0;
      this.percent = 0;
    }
  }